src/useFormState.js

```diff
@@ -246,6 +253,18 @@ export default function useFormState(initialState, options) {
           touch(e);
         }
 
+        let isPristine;
+        if (isFunction(inputOptions.pristinify)) {
```

Comment:
Did not find a better name than pristinify.

Reply:
I think `compare` would be better